### PR TITLE
Tech Preview support for Message tracing and Open Telemetry

### PR DESCRIPTION
**Pull request by kshpak**

### CODE REVIEW

File: QUARKUS-280.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## QUARKUS-280 - Tech Preview support for Message tracing and Open Telemetry
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-280
+
+OpenTelemetry extension for Quarkus is a preferred future solution for the application's telemetry. The current OpenTracing library for telemetry from Smallrye is deprecated and will not be supported in the future.
+Quarkus brings three extenions related to OpenTelemetry:
+ - quarkus-opentelemetry
+ - quarkus-opentelemetry-exporter-jaeger
+ - quarkus-opentelemetry-exporter-otlp
+
```
Comment (rsvoboda) [inline QUARKUS-280.md:10]: Add this text `This feature request is supposed to focus primarily on quarkus-opentelemetry and quarkus-opentelemetry-exporter-jaeger, according to PM they are the demanded by other products.` on new line.

File: QUARKUS-280.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ OpenTelemetry is used to collect the application's telemetry data (metrics, logs, and traces) and to track how a particular user request is traversing through the different stages in a distributed system. Collected data are used to determine application performance and to investigate its behavior.
+
+ Quarkus Guide: https://quarkus.io/guides/opentelemetry  
+ Quarkus Guide upstream: https://github.com/quarkusio/quarkus/commits/main/docs/src/main/asciidoc/opentelemetry.adoc
+
+## Scope of the testing
```
Comment (rsvoboda) [inline QUARKUS-280.md:16]: Our scope of work according the analysis in the rfe is 
AIs for RHBQ 2.2 / QE:
```
    New tests in QE TS to try OpenTelemetry on OpenShift
    Reach equivalent test coverage as for OpenTracing in QE TS
```
Please adjust the TP and link the analysis that I did.

Community side conclusions section says `Community test coverage needs to improve in both Quarkus integration tests and Quarkus Quickstarts (use Jaeger, cover Jaeger exporter, cover Reactive Messaging).`

This effort is for upstream developers, not us. I will create GH issues for these areas as requested by Thomas (https://issues.redhat.com/browse/QUARKUS-280?focusedCommentId=18936836&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18936836  and bellow)

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: QUARKUS-280.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+## QUARKUS-280 - Tech Preview support for Message tracing and Open Telemetry
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-280
+
```
Comment (rsvoboda) [inline QUARKUS-280.md:4]: Add link to the analysis done on RFE - https://issues.redhat.com/browse/QUARKUS-280?focusedCommentId=18936836&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-18936836

**Comment by kshpak** [comment]
PR updated

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**